ocagent.go

```diff
@@ -148,14 +150,17 @@ func (ae *Exporter) Start() error {
 	var err = errAlreadyStarted
 	ae.startOnce.Do(func() {
 		ae.mu.Lock()
-		defer ae.mu.Unlock()
-
 		ae.started = true
 		ae.disconnectedCh = make(chan bool, 1)
 		ae.stopCh = make(chan bool)
 		ae.backgroundConnectionDoneCh = make(chan bool)
+		ae.mu.Unlock()
 
-		ae.setStateDisconnected()
```

Comment:
Let's add a comment here to explain why. This change will block and cause a slow start if WithBlock is set.

Please add this comment:
```go
// An optimistic first connection attempt to ensure that
// applications under heavy load can immediately process
// data. See https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/pull/63
```